resources/models.py

```diff
@@ -101,6 +129,8 @@ class Meta:
 
 
 class Harbor(AbstractArea, TranslatableModel):
+    objects = HarborManager()
```

Comment:
convention is to declare this after all the fields and before the `class Meta`

Reply:
By `this` you mean the `objects/manager`? If so, where can I find more info about it? Since all the model fields are defined before the `class Meta`.

Reply:
yes, I mean `objects = HarborManager()` usually goes after the model fields and before the `class Meta`. [here in the first example](https://docs.djangoproject.com/en/3.0/topics/db/managers/#custom-managers-and-model-inheritance) it is implied and [here in this section and following sections](https://docs.djangoproject.com/en/3.0/topics/db/managers/#custom-managers) it is done exactly like that.

Reply:
here are some examples from wagtail's repo:
https://github.com/wagtail/wagtail/blob/master/wagtail/core/models.py#L73
https://github.com/wagtail/wagtail/blob/master/wagtail/core/models.py#L1594
https://github.com/wagtail/wagtail/blob/master/wagtail/core/models.py#L2187

Reply:
Gotcha 👍 good to know, thanks. 😄